clowdapp.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: floorist
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: floorist
+  spec:
+    database:
+      sharedDbAppName: ${SHARED_DB_APP_NAME}
+    objectStore:
+      - ${BUCKET_NAME}
+    envName: ${ENV_NAME}
+    jobs:
+    - name: metrics-exporter
+      schedule: '*/5 * * * *'
```

Comment:
This should be parameterized, so each project can have its own schedule. Ideally the variable name should be unique enough in *that* big bad repo.

Reply:
I've added a parameter to the template with the surprising name of `SCHEDULE` to configure the cronjob's schedule. However I don't think the variable name needs to be unique - the cronjob definition is supposed to be unique on each Insights app namespace.